CMakeLists.txt

```diff
@@ -55,7 +55,8 @@ endif()
 # to configure the options specify them in the command line or change them in the cmake UI.
 # Don't edit the makefile!
 option(BUILD_SHARED_LIBS "Build shared library" OFF)
-option(CAPSTONE_BUILD_STATIC_RUNTIME "Embed static runtime" ${BUILD_SHARED_LIBS})
+option(BUILD_STATIC_LIBS "Build static library" ON)
+option(BUILD_STATIC_RUNTIME "Embed static MSVC runtime (Windows only). Always set if BUILD_SHARED_LIBS=ON" ${BUILD_SHARED_LIBS})
```

Comment:
Maybe CAPSTONE_BUILD_STATIC_MSVC_RUNTIME?

Reply:
Yes to both. But let's update it in another PR.

Reply:
Because it is not related to the Debian package. I'll open one once this is merged.